src/components/InputTodo.js

```diff
@@ -1,36 +1,45 @@
-import React, { Component } from 'react'
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
 
-export class InputTodo extends Component {
-    state = {
-        title: ''
+class InputTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
     };
+  }
+
+    onChange = (e) => this.setState({ [e.target.name]: e.target.value });
 
-    onChange = e => this.setState({ [e.target.name]: e.target.value, });
-    handleSubmit= e => {
-        e.preventDefault();
-        if (this.state.title.trim()){
-            this.props.addTodoProps(this.state.title);
-            this.setState({ title: '', })
-        } else {
-            alert('Please write item')
-        }
+    handleSubmit= (e) => {
+      e.preventDefault();
+      const { title } = this.state;
+      const { addTodoProps } = this.props;
+      if (title.trim()) {
+        addTodoProps(title);
+        this.setState({ title: '' });
+      } else {
+        // eslint-disable-next-line no-alert
+        alert('Please write item');
+      }
```

Comment:
[OPTIONAL]: It is optional, because the purpose of this exercise was to follow instructions, but in order to follow [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) you should avoid using window.alert which is the same as alert considering the global state of the window object